spec/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/refresher_spec.rb

```diff
@@ -106,6 +107,13 @@ def assert_ems_counts
       expect(ems.storage_manager.cloud_volume_types.count).to eq(2)
     end
 
+    def assert_cloud_manager
+      region = Rails.application.secrets.ibm_cloud_power[:ibmcloud_region]
```

Comment:
Ah nevermind

Reply:
Yeah, the `ibmcloud_region` value is also used throughout the terraform automation to create our spec environment.